zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/install/InstallInterpreter.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.zeppelin.interpreter.install;
+
+import org.apache.log4j.ConsoleAppender;
+import org.apache.log4j.Logger;
+import org.apache.zeppelin.conf.ZeppelinConfiguration;
+import org.apache.zeppelin.dep.DependencyResolver;
+import org.apache.zeppelin.util.Util;
+import org.sonatype.aether.RepositoryException;
+import java.io.File;
+import java.io.IOException;
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * Commandline utility to install interpreter from maven repository
+ */
+public class InstallInterpreter {
+  static ZeppelinConfiguration conf = ZeppelinConfiguration.create();
+
+  static String [] availableInterpreters = {
+    "md",     "zeppelin-markdown",   "Markdown interpreter",
+    "sh",     "zeppelin-shell",      "Allows shell command",
+    "jdbc",   "zeppelin-jdbc",
+    "Generic JDBC interpreter for hive, phoenix, postsgresql, mysql, etc",
```

Comment:
do you think we would need to update this list in the future?
is there a way to have a text lookup table with a list of short name instead of hardcoding it in .java file?